sdk/connectedvmwarevsphere/Azure.ResourceManager.ConnectedVMwarevSphere/tests/Azure.ResourceManager.ConnectedVMwarevSphere.Tests.csproj

```diff
@@ -1,5 +1,13 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <ItemGroup>
+    <ProjectReference Include="..\..\..\core\Azure.Core\src\Azure.Core.csproj" />
```

Comment:
Usually we should not explicitly include this as a project reference. Is there any reason for this?

Reply:
This was not included explicitly hence I was getting error, hence included it manually.